Resources/Textures/DeltaV/Objects/Weapons/Guns/Projectiles/projectiles.rsi/meta.json

```diff
@@ -0,0 +1,56 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Modified by Mike from cev-eris at https://github.com/discordia-space/CEV-Eris/tree/7ff8f28ea4de734f3cc3cb70f2d4e4b4263a988d",
```

Comment:
Mike who? Where are they?

Reply:
Mike is the pr author, @musicmanvr

Reply:
Nobody else is going to know that from looking at the meta file